File: LE6/d8-2.java
import java.util.Scanner;

public class Ex8_V2 {

    public static String classificar_triangulo(double a, double b, double c) {
        if (a == b && b == c) {
            return "Equilátero";
        } else if (a == b || a == c || b == c) {
            return "Isósceles";
        } else {
            return "Escaleno";
        }
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        double a, b, c;

        System.out.print("Digite o lado 1: ");
        a = sc.nextDouble();

        System.out.print("Digite o lado 2: ");
        b = sc.nextDouble();

        System.out.print("Digite o lado 3: ");
        c = sc.nextDouble();

        String resultado = classificar_triangulo(a, b, c);

        System.out.println("Triângulo " + resultado);
    }
}

File: LE6/d8-2.py
def classificar_triangulo(a, b, c):
    if a == b and b == c:
        return "Equilátero"
    elif a == b or a == c or b == c:
        return "Isósceles"
    else:
        return "Escaleno"

a = float(input("Digite o lado 1: "))
b = float(input("Digite o lado 2: "))
c = float(input("Digite o lado 3: "))

resultado = classificar_triangulo(a, b, c)
print("Triângulo", resultado)